src/agent/MessageSender.ts

```diff
@@ -9,19 +9,19 @@ import { TransportService } from './TransportService'
 export class MessageSender {
   private envelopeService: EnvelopeService
   private transportService: TransportService
-  private outboundTransporter?: OutboundTransporter
+  private _outboundTransporter?: OutboundTransporter
 
   public constructor(envelopeService: EnvelopeService, transportService: TransportService) {
     this.envelopeService = envelopeService
     this.transportService = transportService
   }
 
   public setOutboundTransporter(outboundTransporter: OutboundTransporter) {
```

Comment:
Generally I'm in favour of `get XXX` over `getXXX` if there are no side effects.

I didn't change the `setXXX` as It may feel weird to set the transport by doing `agent.outboundTransporter = XXX` over `agent.setOutboundTransporter(XXX)`

Reply:
I see. Yeah, that was exactly the reason I kept getXxx and setXxx for both 😄 . But I'm good with it. API will probably change with multi-transports anyway